### ISSUE TITLE
Edge - OSPF routing options don't work

### ISSUE THREAD

**Issue by pandom** [vmware/powernsx#196]
When using -gracefulrestart or -defaultoriginate the commands are not being executed. 

**TEMPORARY WORKAROUND**
```
$edge = get-nsxedge edge01
$edge.features.routing.ospf.defaultOriginate = "true"
$edge.features.routing.ospf.gracefulRestart = "true"
$edge | set-nsxedge -confirm:$false | out-null
```

**Comment by koh39** [comment]
Same applied for DLR/UDLR
- Non working code
Get-NsxLogicalRouter $Udlr1Name | Get-NsxLogicalRouterRouting | Set-NsxLogicalRouterOspf -GracefulRestart:$false -confirm:$false | out-null

- WA is slightly different since Set-NsxLogicalRouter doesn't exist
$Udlr1Tmp = Get-NsxLogicalRouter $Udlr1Name | Get-NsxLogicalRouterRouting
$Udlr1Tmp.ospf.gracefulRestart = "false"
$Udlr1Tmp | Set-NsxLogicalRouterRouting -confirm:$false | out-null

**Comment by pandom** [comment]
FYI 

http://pubs.vmware.com/Release_Notes/en/nsx/6.3.2/releasenotes_nsx_vsphere_632.html

defaultOriginate parameter removed from the following methods for logical (distributed) router NSX Edge appliances only:

> GET/PUT /api/4.0/edges/{edge-id}/routing/config/ospf
> GET/PUT /api/4.0/edges/{edge-id}/routing/config/bgp
> GET/PUT /api/4.0/edges/{edge-id}/routing/config

This explains why the cmdlet doesn’t function correctly anymore as discussed here: https://github.com/vmware/powernsx/issues/196

**Comment by nmbradford** [comment]
merged via #342
